samples/net/sockets/raw/README.rst

```diff
@@ -0,0 +1,27 @@
+.. _raw-socket-sample:
+
+Raw socket sample
+#################
+
+Overview
+********
+
+This sample is a simple RAW socket application showing usage of
```

Comment:
Ethernet is a registered trademark so it's always capitalized.

Let's change this paragraph to fix grammar and spelling:
```
This sample is a simple application showing usage of
raw sockets over Ethernet.  This sample prints every Ethernet
packet received and sent over the wire. The Zephyr
network subsystem does not touch any of the headers (e.g., L2, L3). 
```